### PR TITLE
Fix dependency

### PR DESCRIPTION
**Pull request by stephenc**

### CODE REVIEW

**Comment by jglick** [comment]
This is for https://github.com/jenkinsci/analysis-core-plugin/pull/31 I suppose.

**Comment by amuniz** [comment]
@jglick yes

File: pom.xml [hunk 1]
```diff
@@ -92,13 +92,6 @@
         <groupId>com.infradna.tool</groupId>
         <artifactId>bridge-method-injector</artifactId>
         <version>1.14</version>
-        <executions>
-          <execution>
-            <goals>
-              <goal>process</goal>
-            </goals>
-          </execution>
-        </executions>
```
Comment (amuniz) [inline pom.xml:101]: Nop. The execution is needed. Otherwise the goal is never invoked in the lifecycle.

**Comment by amuniz** [comment]
I merged this manually. Closing.
Thanks.

**Pull request closed without merging**